pallets/aleph/src/lib.rs

```diff
@@ -287,6 +294,12 @@ pub mod pallet {
                 false => Ok(()),
             }
         }
+
+        pub fn verify_multisignature(msg: &Vec<u8>, sgn: &SignatureSet<Signature<T>>) -> bool {
+            let authority_verifier =
+                primitives::crypto::AuthorityVerifier::new(Self::authorities());
+            primitives::crypto::AuthorityVerifier::is_complete(&authority_verifier, msg, sgn)
```

Comment:
Not sure whether this is also the case in pallet code, some conventions here are different, but in case it is pointing another instance out.

Reply:
This is a leftover, imported on module level.